packages/site-server/package.json

```diff
@@ -39,6 +41,12 @@
       "dependencies": [
         "build"
       ]
+    },
+    "serve:prod": {
+      "command": "node --enable-source-maps ./lib/prod-server.js",
```

Comment:
I think we already have this script in the top-level. I'm ok with it being here instead, but I think we should remove it from the top-level then.

Reply:
k